### PR TITLE
Conventions fixes + no more home menu

### PR DESCRIPTION
**Pull request by PhoenixBound**
This makes things a bit more consistent, but I haven't tried Mocha or WUP Installer Mod to check some of the changes.

Some of these changes might have to be made to the 3DS guide as well, so I'll whip up another pull request for that.

### CODE REVIEW

**Comment by curtisy1** [comment]
I agree to most of those, but "hombebrew" should actually be fixed(?) also vWii Wii menu sounds kind of repetitive so I would call it vWii menu only, but that's just my view on this. 

Still great job on finding some more typos! Keep it up as I won't be able to commit much from now on :)

**Comment by PhoenixBound** [comment]
Fixed the part about vWii menu (also found another typo, so I fixed that too).

Could you elaborate about the part with "hombebrew"? I didn't catch what you meant by that.

**Comment by curtisy1** [comment]
Never mind, I actually fixed the typo in line 50 of get started.txt
(hombebrew) two days ago, but I just saw it was included again so sorry for
the confusion. Seeing this though, would you mind comparing the current
master with my previous pull requests and see if some mistakes were
reintroduced? Would really help me out if you did.

Thanks!

**Comment by PhoenixBound** [comment]
I think I found one possible error, but it's a little confusing and inconsistent so I'll just explain the whole thing here:

* The guide refers to wupinstaller as of this PR as *Modified WUP Installer* in Get-Started.txt and *WUP Installer Mod* in the other two pages.
* The release page for the tool calls it "WUP Installer y Mod {version}."
* The repository's readme calls it by its full original name, "WiiU public title installer."
* The HBL meta.xml file calls it "WUP Installer - Mod Y"
* And the app itself just calls it "WUP Installer" with a separate mod notice later.

One instance of "WUP Installer - Mod Y" was originally in the guide and taken out, which was how I noticed it at first.

For now, I'm just going to rename them to what they call themselves at the time they're being mentioned. Get-Started.txt will have "WUP Installer y Mod," and the other two pages will have "WUP Installer - Mod Y."

File: _pages/en_US/Home.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ permalink: /index.html
 
 ## What is Homebrew?
 
-[**Homebrew**](https://en.wikipedia.org/wiki/List_of_homebrew_video_games) usually refers to software that is not authorized by Nintendo. It allows you to run homebrew games, tools like save editing and backup, and emulators for various older systems.
+[**Homebrew**](https://en.wikipedia.org/wiki/List_of_homebrew_video_games) usually refers to software that is not authorized by Nintendo. It allows you to run homebrew games, tools like save editors or backup software, and emulators for various older systems.
```
Comment (Plailect) [inline _pages/en_US/Home.txt:31]: No, this is for actual save backups. This is NOT a reference to "backups".

File: _pages/en_US/Mocha CFW (Haxchi).txt [hunk 2]
```diff
@@ -76,7 +76,7 @@ You can use the (Home) button to exit the Homebrew Launcher if you need to purch
   + Use syshax.xml (coldboothax): **off**
 11. Press (A) to confirm and exit
 12. Your Wii U should display a black screen; this is the RedNAND setup
-13. Press the power button **on the console** (not the gamepad) to start the copy of your internal system memory to your SD card
+13. Press the power button **on the console** (not the gamepad) to start copying your internal system memory to your SD card
```
Comment (Plailect) [inline _pages/en_US/Mocha CFW (Haxchi).txt:79]: This change is incorrect.

File: _pages/en_US/Mocha CFW (Homebrew Launcher).txt [hunk 3]
```diff
@@ -35,7 +35,7 @@ In order to boot the patched signature checks and boot RedNAND, you will need to
   + Use syshax.xml (coldboothax): **off**
 8. Press (A) to confirm and exit
 9. Your Wii U should display a black screen; this is the RedNAND setup
-10. Press the power button **on the console** (not the gamepad) to start the copy of your internal system memory to your SD card
+10. Press the power button **on the console** (not the gamepad) to start copying your internal system memory to your SD card
```
Comment (Plailect) [inline _pages/en_US/Mocha CFW (Homebrew Launcher).txt:38]: This change is incorrect.

File: _pages/en_US/Mocha CFW (CBHC).txt [hunk 4]
```diff
@@ -4,7 +4,7 @@ lang: en
 permalink: /mocha-cfw-(cbhc).html
 ---
 
-CBHC (Coldboot Haxchi) changes your default system title to make your SysNAND launch the DS virtual console game by default instead of the home menu, which will then load RedNAND.
+CBHC (Coldboot Haxchi) changes your default system title to make your SysNAND launch the DS virtual console game by default instead of the Wii U Menu, which will then load RedNAND.
```
Comment (Plailect) [inline _pages/en_US/Mocha CFW (CBHC).txt:7]: "home menu" is a much better descriptive term that is more easily identifiable

File: _pages/en_US/Mocha CFW (Haxchi).txt (hunk 2 above)
Comment (PhoenixBound) [reply _pages/en_US/Mocha CFW (Haxchi).txt:79]: Whoops, I'll revert that then.

I'm a little confused about what it means to "start" a copy. Is it like "starting up"?

**Comment by Plailect** [comment]
https://github.com/Plailect/Guide_WiiU/commit/4b0cee6abfa743a4876f881446b1922f7e53d65e

**Pull request closed without merging**